packit/cli/copr_build.py

```diff
@@ -57,6 +58,27 @@
     help="Installation instructions for the project to build in.",
     default=None,
 )
+@click.option(
+    "--listed-on-hp",
+    help="Created copr project will be visible on copr's home-page.",
+    default=False,
+    is_flag=True,
+)
+@click.option(
+    "--preserve-project",
```

Comment:
Could an option taking the number of days to preserve the project offer more flexibility? `0` could mean "preserve forever" while the default could be `60`.

Reply:
Tha will also save us some problems with overwriting since I am not sure (now) how I can unset the deletion.

But is it useful for users? I would say that they want to either preserve or remove automatically the project. Do they care about specific numbers?

Reply:
Why do we need to unset deletion?

Yeah, you have a point, maybe users don't care about the numbers that much, and if they do, it might be better to control those numbers in a separate option.

Reply:
> Why do we need to unset deletion?

It can be useful when they don't know about that option in the time when the project is created or the option is not present (e.g. our project for master builds of packit has it said and we can't edit that via packit.)

Reply:
Based on the current help text I would expect expiration to be set only when the project is created by Packit:

> Created copr project will not be removed after 60 days.

So when the Copr project already exists, I would expect this flag to have no effect.

Reply:
Ok, this will be tricky since no-value and default-value cannot be easily differentiated...